docs/specification/README.md

```diff
@@ -440,17 +461,21 @@ A writer may choose to put messages in Chunks to compress record data. This MCAP
 
 ```
 [Header]
+[Datatype]
```

Comment:
The other message examples use ids. It will get a bit confusing to use another sequence of ids for Datatype, but I think it's even more confusing to leave them out.

Reply:
Added some identifiers here A, B